java-checks/src/main/java/org/sonar/java/checks/helpers/QuickFixHelper.java

```diff
@@ -102,6 +113,72 @@ public static SyntaxToken previousToken(Tree tree) {
     return previousToken(parent);
   }
 
+  /**
+   * Returns the preceding variable in a mutli-variable declaration.
+   *
+   * @param current The target variable
+   * @return An Optional with the preceding variable, Optional.empty otherwise.
+   */
+  public static Optional<VariableTree> previousVariable(VariableTree current) {
+    Tree parent = current.parent();
+    if (parent.is(Tree.Kind.BLOCK, Tree.Kind.INITIALIZER, Tree.Kind.STATIC_INITIALIZER)) {
+      List<Tree> children = ((BlockTreeImpl) parent).children();
+      return previousVariable(current, children, TRIMMING_OFFSET_IN_BLOCK_CHILDREN);
+    } else if (parent.is(Tree.Kind.CLASS)) {
```

Comment:
There's not only CLASS but ENUM, INTERFACE, ANNOTATION_TYPE, RECORD (all instances of ClassTree)
And for RECORD the variable can be in `recordComponents()` or `members()`. If the variable is in `recordComponents()`, it has no previous variable or next variable.